### PR TITLE
Ensure `disabled` attr actually disables the button

### LINKED ISSUE #70: [BUG] - `disabled` assign not respected in `RegistrationComponent`

### PR DESCRIPTION
**Pull request by peaceful-james**
Resolves #70

### CODE REVIEW

**Comment by peaceful-james** [comment]
I will start with an easy one :)

**Comment by peaceful-james** [comment]
@type1fool I snuck it a small extra change. I stopped disabling the sign in button. This should never be disabled.

File: templates/live_views/authentication_live.html.heex [hunk 1]
```diff
@@ -39,7 +39,7 @@
       <p>Sign into an existing account:</p>
 
       <.live_component
-        disabled={@form.source.valid? == false}
+        disabled={false}
```
Comment (type1fool) [inline templates/live_views/authentication_live.html.heex:42]: Good catch.

File: lib/webauthn_components/base_components.ex [hunk 2]
```diff
@@ -13,7 +13,7 @@ defmodule WebauthnComponents.BaseComponents do
   @doc false
   def button(assigns) do
     ~H"""
-    <button id={@id} type={@type} {@rest}>
+    <button id={@id} type={@type} {@rest} {if @disabled, do: %{disabled: true}, else: %{}}>
```
Comment (type1fool) [inline lib/webauthn_components/base_components.ex]: This syntax accomplishes the same result, and it's more idiomatic for component assigns.

```suggestion
    <button id={@id} type={@type} disabled={@disabled} {@rest}>
```

Comment (type1fool) [reply lib/webauthn_components/base_components.ex]: I pushed dcd6613 for this.

**Review by type1fool — APPROVED** [review]
Thank you @peaceful-james for catching this and opening a PR. I added a couple commits to make some small adjustments, and merged some CI fixes from `main`.

File: lib/webauthn_components/base_components.ex (hunk 2 above)
Comment (peaceful-james) [reply lib/webauthn_components/base_components.ex]: This works, which is surprising for me.

I know `disabled` matches CSS selectors simply by being present on the element.

However, the attribute is simply not present for `disabled={false}` (at least, on my version of LiveView), which is fantastic news.

**Pull request merged**